core/src/tx_sender/mod.rs

```diff
@@ -110,41 +114,87 @@ impl TxSender {
         }
     }
 
-    /// Gets the current recommended fee rate from the Bitcoin Core node.
-    ///
-    /// Uses the `estimatesmartfee` RPC call with a confirmation target of 1 block
-    /// and conservative estimation mode.
-    ///
-    /// If fee estimation is unavailable (e.g., node is warming up), it returns
-    /// an error, except in Regtest mode where it defaults to 1 sat/vB for testing convenience.
-    async fn _get_fee_rate(&self) -> Result<FeeRate> {
-        tracing::info!("Getting fee rate");
-        let fee_rate = self
-            .rpc
-            .client
-            .estimate_smart_fee(1, Some(EstimateMode::Conservative))
-            .await;
-
-        match fee_rate {
-            Ok(fee_rate) => match fee_rate.fee_rate {
-                Some(fee_rate) => Ok(FeeRate::from_sat_per_kwu(fee_rate.to_sat())),
-                None => {
-                    if self.paramset.network == bitcoin::Network::Regtest {
-                        tracing::debug!("Using fee rate of 1 sat/vb (Regtest mode)");
-                        return Ok(FeeRate::from_sat_per_vb_unchecked(1));
-                    }
-
-                    tracing::error!("TXSENDER: Fee estimation error: {:?}", fee_rate.errors);
-                    Ok(FeeRate::from_sat_per_vb_unchecked(1))
-                }
-            },
-            Err(e) => {
-                tracing::error!("TXSENDER: Error getting fee rate: {:?}", e);
+    /// Gets the current recommended fee rate in sat/vb from Mempool Space or Bitcoin Core.
+    async fn _get_fee_rate_as_sat_vb(&self, config: &BridgeConfig) -> Result<FeeRate> {
+        match self.paramset.network {
+            // Regtest and Signet use a fixed, low fee rate.
+            Network::Regtest | Network::Signet => {
+                tracing::info!(
```

Comment:
debug might be better here. dont want to clutter the info logs with fee info every time we run this